### ISSUE TITLE
Lightsail Instance Hosed

### ISSUE THREAD

**Issue by scottholmes** [bitnami/vms#50]
### Platform

AWS

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

771f9880-bf57-2d62-f411-ecb037f5a608

### bndiagnostic output

\-Error copying file from /opt/bitnami/php/logs/php-fpm.log to /tmp/bndiagnostic-b0f8e4ac1
fad9810/bitnami-lampstack-info-20220629-182800-1160/php/logs
:/opt/bitnami/php/logs/php-fpm.log is not readable
Abort, Retry, Ignore ? [A/r/i]r

Error copying file from /opt/bitnami/php/logs/php-fpm.log to /tmp/bndiagnostic-b0f8e4ac1fa
d9810/bitnami-lampstack-info-20220629-182800-1160/php/logs
:/opt/bitnami/php/logs/php-fpm.log is not readable
Abort, Retry, Ignore ? [A/r/i]a

The diagnostic bundle was uploaded successfully to the Bitnami servers. Please copy the fo
llowing code:

771f9880-bf57-2d62-f411-ecb037f5a608

And paste it in your Bitnami Support ticket.

### bndiagnostic was not useful. Could you please tell us why?

I recieved no suggestion from tool

### Describe your issue as much as you can

I hosed my site attempting to allow https.  I would like to set the instance back to its default configuration and just run http.  I tried diableing the https port to no avail.  The site is a lightsail instance.

**Comment by scottholmes** [comment]
I tried bndiagnostics again and again it reported errors.. Here is the resulting bundle code:  e9b5d44e-65ff-48cb-d451-ca0011062561

**Comment by mdhont** [comment]
The following guide shows how to force HTTPS. You need to undo the changes mentioned in it:

https://docs.bitnami.com/aws/apps/wordpress/administration/force-https-apache/

**Comment by scottholmes** [comment]
Thanks for the reply.  I decided to just start over.  I created a new instance, not from a snapshot, and populated it with my website code backed up on my development computer.  My site is a drupal 9-4 created by composer, not the prepackaged setup provided by AWS.  My apologies for difficulties in my not separating the AWS factors from the bitnami factors.  It all looked to same to me until the AWS service reps told me they can't help with bitnami.
Thanks again for the reply.
